ci/vcpkg/vcpkg.json

```diff
@@ -20,6 +20,7 @@
     "flatbuffers",
     "gflags",
     "glog",
+    "libxml2",
```

Comment:
Can we create `azure` feature and move this to the feature?

Reply:
Done